src/DssSpell.sol

```diff
@@ -59,106 +44,41 @@ contract SpellAction {
     //
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
-    uint256 constant FOUR_PERCENT_RATE = 1000000001243680656318820312;
+//    uint256 constant ZERO_PERCENT_RATE = 1000000000000000000000000000;
```

Comment:
nit. these comments are not aligned but can probably be removed

Reply:
Oh I just had them in there to cover all bases since the outcome of the stability fee poll was uncertain, already removed.